### PR TITLE
Add RawRepresentable defaults for Strings and Ints

### PR DESCRIPTION
**Pull request by tonyd256**

### CODE REVIEW

File: Argo/Extensions/RawRepresentable.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    default: return .typeMismatch("String", actual: json)
+    }
+  }
+}
+
+public extension Decodable where Self.DecodedType == Self, Self: RawRepresentable, Self.RawValue == Int {
+  static func decode(json: JSON) -> Decoded<Self> {
+    switch json {
+    case let .Number(n): return .fromOptional(self.init(rawValue: n as Int))
+    default: return .typeMismatch("\(self)", actual: json)
```
Comment (gfontenot) [inline Argo/Extensions/RawRepresentable.swift]: Shouldn't this be `Int` to match the String version?

Comment (tonyd256) [inline Argo/Extensions/RawRepresentable.swift]: yup good catch

**Comment by gfontenot** [comment]
~[](http://dl.dropboxusercontent.com/u/452364/gifs/spidey-dance.gif)

**Comment by swizzlr** [comment]
Sorry I dropped the ball on this guys – haven't had much non-development time to go around.

**Comment by gfontenot** [comment]
No worries, @swizzlr. I know that feeling.

**Comment by tonyd256** [comment]
Definitely no worries! Thanks for starting it and bringing it to our attention! It's a big win for Argo!

**Pull request merged**